packages/build-tools/src/steps/functions/repack.ts

```diff
@@ -148,6 +160,25 @@ export function createRepackBuildFunction(): BuildFunction {
   });
 }
 
+/**
+ * Install `@expo/repack-app` in a sandbox directory and import it.
+ */
+async function installAndImportRepackAsync({
+  version = 'latest',
+  tmpDir,
+}: {
+  version?: string;
+  tmpDir: string;
+}): Promise<typeof import('@expo/repack-app')> {
+  const sandbox = path.join(tmpDir, 'packageRoot');
```

Comment:
Maybe it would be safer if this function created its own `tmpDir`? Just in case in the future we start using `tmpDir` for something new that suddenly ends up in the same Yarn workspace or something.

Reply:
good point. done in the latest commit